addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/handler/server/MinecraftSocketHandler.java

```diff
@@ -51,7 +51,7 @@ public void onClose() {
 
     @Override
     public void onError(WebSocketException e) {
-        logger.error("Server error " + e.getMessage() + " : " + e.getCause());
+        logger.error("Server error {} : {}", e.getMessage(), e.getCause());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace